unittests/test_jira_import_and_pushing_api.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, *args, **kwargs):
         DojoVCRAPITestCase.__init__(self, *args, **kwargs)
 
     def assert_cassette_played(self):
-        if True:  # set to True when committing. set to False when recording new test cassettes
+        if False:  # set to True when committing. set to False when recording new test cassettes
```

Comment:
Should this be False?

Reply:
Good catch! it should be set to `True`
```suggestion
        if True:  # set to True when committing. set to False when recording new test cassettes
```